bgpd/bgp_attr_evpn.c

```diff
@@ -0,0 +1,262 @@
+/* Ethernet-VPN Attribute handling file
```

Comment:
I just saw that a similar function ( mac2str) was put in evpn branch in prefix.c.
if everybody agrees, I can move  some functions ( not all) in lib/<filenameto specify>

convertchartohexa
str2mac
str2esi
esi2str
mac2str